INSTALL.md

```diff
@@ -1560,7 +1560,7 @@ Configuration Problems
 The `./config` script tries hard to guess your operating system, but in some
 cases it does not succeed. You will see a message like the following:
 
-    $ ./config
+    ./config
     Operating system: x86-whatever-minix
     This system (minix) is not supported. See file INSTALL for details.
```

Comment:
Damnit, I didn't pay enough attention.

A reason for the `$` was to clearly mark what is prompted commands and what is command output.   It's even explained here (which, interestingly enough, you ignored in all kinds of ways):

https://github.com/openssl/openssl/blob/922f156545c9c8397abbda49eb640f7a34969231/INSTALL.md#L73-L81

Reply:
I find it *incredibly irritating* that things like this get "slipped in" in PRs that have nothing to do with it.

Reply:
Indeed I overlooked that explanation, but I did suspect that the `$ ` was used for that purpose.

Still I don't like it, because in particular if there is a number of shell commands in a row, such as in
```
    $ mkdir /var/tmp/openssl-build
    $ cd /var/tmp/openssl-build
    $ whatever
```
you cannot simply copy&paste that series of commands, while with
```
    mkdir /var/tmp/openssl-build
    cd /var/tmp/openssl-build
    whatever
```
you can.

And I suppose there are just a few cases where command input and output could be confused.
In such cases we could, e.g., insert a blank line to help understanding this distinction.

Reply:
Regardless, if you want to change that, having that "slip in" in a PR that's about something different is not a good way to do it.

Reply:
> I find it _incredibly irritating_ that things like this get "slipped in" in PRs that have nothing to do with it.

Yeah, I should better have done this in a separate PR for clarity.
It just had come up along with the review of this PR, so for simplicity I added a commit for that here.

Reply:
> Regardless, if you want to change that, having that "slip in" in a PR that's about something different is not a good way to do it.

Sorry for that.

Reply:
I've just checked: in this (merged) PR this is the only case where command input and output got mixed.
Here suggest to simply drop the line
```
    ./config
```
because it is already mentioned in the sentence above it.

@levitte, are you okay if I do this nit fix along with #12109 or #12232,
or should I also in this case do a separate PR for it?

Reply:
And of course we should be consistent about using `$ ` or not,
and if there is consensus to drop it also remove the remark in `INSTALL.md` on its purpose quoted above.

Reply:
BTW, the `$` symbols is not commonly used under Windows, where typically a `>` is printed as prompt.

Reply:
I just realized that already today at noon (CEST) a PR restoring the `$ ` has been done: #12257.